client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -201,6 +201,20 @@ Undocumented.prototype.jetpackAuthorizeSSONonce = function( siteId, ssoNonce, fn
 	return this.wpcom.req.post( { path: endpointUrl }, params, fn );
 };
 
+Undocumented.prototype.getJetpackSyncStatus = function( siteId, fn ) {
+	debug( '/sites/:site_id:/sync/status query' );
+	const endpointUrl = '/sites/' + siteId + '/sync/status';
+	const params = {};
+	return this.wpcom.req.get( { path: endpointUrl }, params, fn );
```

Comment:
I think this could be a bit simplefied. 
```
Undocumented.prototype.getJetpackSyncStatus = function( siteId, fn ) {
    debug( '/sites/:site_id:/sync/status query' );
    return this.wpcom.req.get( '/sites/' + siteId + '/sync/status'', fn );
}
``` 
See https://github.com/Automattic/wpcom.js/blob/master/lib/util/request.js#L22